src/quanteda/plot_missing_vals.py

```diff
@@ -0,0 +1,24 @@
+import altair as alt
+import pandas as pd
+
+def plot_missing_vals():
```

Comment:
You have to include the parameters in the function call here
```suggestion
def plot_missing_vals(df):
```